src/store/ducks/plans.ts

```diff
@@ -657,6 +660,11 @@ export const makePlansArraySelector = (planKey?: string) => {
       getPlansArrayByParentJurisdictionId(planKey),
     ],
     (plans, plans2, plans3, plans4, plans5) =>
-      intersect([plans, plans2, plans3, plans4, plans5], JSON.stringify)
+      SORT_FIELD
+        ? reverseChronologicalSort(
+            intersect([plans, plans2, plans3, plans4, plans5], JSON.stringify),
+            SORT_FIELD
+          )
+        : intersect([plans, plans2, plans3, plans4, plans5], JSON.stringify)
```

Comment:
This violates the [closed/open principle](https://stackify.com/solid-design-open-closed-principle/).  That is, if we ever wanted to sort by some other field we would need to modify the code instead of just changing a parameter or something similar.

Is it possible to make the sort field an optional param to `makePlansArraySelector` and get it from there?